eng/liveILLink.targets

```diff
@@ -2,7 +2,7 @@
 
   <!-- NOTE: these targets should be a no-op, unless _RequiresLiveILLink, computed below, is true. -->
 
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
```

Comment:
This is incorrect. We have several projects that target i.e. `$(NetCoreAppCurrent)-windows` or similar. You want to use the `IsTargetFrameworkCompatible` function here.